zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala

```diff
@@ -45,14 +44,18 @@ trait CassandraSpanStoreFactory {self: App =>
     new CassandraSpanStore(
       repository,
       stats.scope(keyspace()),
-      cassieSpanTtl(),
-      cassieIndexTtl(),
-      cassieMaxTraceCols())
+      cassandraSpanTtl(),
+      cassandraIndexTtl(),
+      cassandraMaxTraceCols())
   }
 
   def createClusterBuilder(): Cluster.Builder = {
-    addContactPoint(Cluster.builder())
-      .withRetryPolicy(ZipkinRetryPolicy.INSTANCE)
+    val builder = if(!cassandraUser.isDefined || !cassandraPassword.isDefined)
```

Comment:
I think this would be clearer to first initalize, then guard entry to withCredentials.

ex. 
```
val builder = addContactPoint(Cluster.builder())
if(!cassandraUser.isDefined || !cassandraPassword.isDefined) {
  builder.withCredentials(cassandraUser(), cassandraPassword())
}
```